benchmark/scripts/Benchmark_Driver

```diff
@@ -90,9 +90,17 @@ def instrument_test(driver_path, test, num_samples):
     test_output = (test_output_raw.split()[1].split(',') + [peak_memory])
     return test_output
 
-def get_tests(driver_path):
+
+def get_tests(driver_path, args):
     """Return a list of available performance tests"""
-    return subprocess.check_output([driver_path, '--list']).split()[2:]
+    tests = subprocess.check_output([driver_path, '--list']).split()[2:]
+    if args.filter:
```

Comment:
Can you use early returns here to eliminate the elif. You do not have any code of consequence after the condition. This is communicated to the reader of the code in a more succint way by using early returns. i.e.:

```
if args.filter:
    prefix = args.filter
    return filter(lambda name: name.startswith(prefix), tests)

if not args.benchmark:
    return tests

benchmarks = set(args.benchmark)
return sorted(list(set(tests).intersection(benchmarks)))
```

Notice how just by reading the code the reader is able to know the amount of work left to be returned without having to read the whole function.